scribejava-apis/src/main/java/com/github/scribejava/apis/FlickrApi.java

```diff
@@ -34,7 +44,13 @@ public String getAccessTokenEndpoint() {
      */
     @Override
     public String getAuthorizationUrl(OAuth1RequestToken requestToken) {
-        return "https://www.flickr.com/services/oauth/authorize?oauth_token=" + requestToken.getToken();
+        String authUrl = String.format(AUTHORIZE_URL, requestToken.getToken());
+
+        if (perm != null) {
+            authUrl += "&perms=" + perm;
+        }
+
+        return authUrl;
```

Comment:
Seems, I'm wrong. disregard my comment, pls.
Your PullRequst looks ok, I will return to it a bit later